en/components/calendar.md

```diff
@@ -4,40 +4,190 @@ _description: Create an intuitive calendar for applications to display date info
+
 ```typescript
-import { IgxCalendarComponent } from "igniteui-angular";
+// app.module.ts
+...
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { IgxCalendarModule } from 'igniteui-angular/main';
+@NgModule({
+    ...
+    imports: [..., BrowserAnimationsModule, IgxCalendarModule],
+    ...
+})
+export class AppModule {}
+```
+You will usually also import the **IgxCalendarComponent** in the AppComponent file (or your editor will auto-import them for you) when declaring types that are part of the calendar API:
+.
+
+```typescript
+import { IgxCalendarComponent } from 'igniteui-angular/main';
+...
+
+@ViewChild('calendar', { read: IgxCalendarComponent }) public calendar: IgxCalendarComponent;
 ```
 
-Basic initialization
+> [!WARNING]
+> Note that the **igxCaledarComponent** uses the [Intl](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat) WebAPI for localization and formatting of dates. 
+> Consider using the [appropriate polyfills](https://github.com/andyearnshaw/Intl.js/) if your target platform does not support them.
+
+### Selection
+Instantiating the **IgxCalendarComponent** is as easy as placing its selector element in the template. This will leave the calendar display the current month by default and run in single selection mode. We switch to any of the other selection modes - **multi** and **range**, by setting value to the **selection** property:
 ```html
+<!-- app.component.html -->
+<!-- Single selection mode -->
 <igx-calendar></igx-calendar>
+<!-- Multi selection mode -->
+<igx-calendar selection="multi"></igx-calendar>
+<!-- Range selection mode -->
+<igx-calendar selection="range"></igx-calendar>
 ```
+Notice that the calendar header is not rendered when the selection is either `multi` or `range`:
+<div class="sample-container" style="height: 500px">
+    <iframe id="calendar-sample-1-iframe" src='{environment:demosBaseUrl}/calendar-sample-1' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="calendar-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+###Localization and formatting
+
+Due to their very nature, localization and formatting are essentials to any calendar. In the **IgxCalendarComponent** those are controlled and customized through the following properties - **locale**, **formatOptions**, **formatViews**:
+
+| Name | Description |
+|:--:|:---|:---|
+| `locale` | Controls the locale used for formatting and displaying the dates in the calendar. The expected string value should be a [BCP 47 language tag](https://tools.ietf.org/html/rfc5646) and if not set defaults to en-EN. |
```

Comment:
I've never seen the culture code en-EN.  The first part is the neutral culture and the second part indicates a specific country.  There is no "EN" country.  Could be "US" or "GB"?  Or is this completely different than the other culture codes?

Reply:
Ex. table here: https://msdn.microsoft.com/en-us/library/system.globalization.cultureinfo(v=vs.71).aspx

Reply:
I have updated the article to mention that it defaults to "en" and opened new issue so we fix this in code. https://github.com/IgniteUI/igniteui-angular/issues/653